test/catch2/live_restore/live_restore_test_env.cpp

```diff
@@ -60,4 +60,11 @@ live_restore_test_env::source_file_path(const std::string &file_name)
     return DB_SOURCE + "/" + file_name;
 }
 
+std::string
+live_restore_test_env::tombstone_file_path(const std::string &file_name)
+{
+    // Tombstone files only exist in the destination folder.
+    return DB_DEST + "/" + file_name + ".deleted";
```

Comment:
```suggestion
    return DB_DEST + "/" + file_name + WT_LIVE_RESTORE_FS_TOMBSTONE_SUFFIX;
```
Not that sure if we use MACRO that is defined in src, if so I think we can replace `".deleted"` with the MACRO here.

Reply:
We do this elsewhere so its okay to use the src code macro.

Reply:
[0b2094c](https://github.com/wiredtiger/wiredtiger/pull/11369/commits/0b2094cd02acfd24efa9be4bb7546526ad980e02)